be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.en.resx

```diff
@@ -318,4 +318,46 @@
   <data name="NewClientAddress" xml:space="preserve">
     <value>Add address</value>
   </data>
+  <data name="FieldDefinitionNotFound" xml:space="preserve">
+    <value>Field definition not found</value>
+  </data>
+  <data name="FieldDefinitionSelectTypeConflict" xml:space="preserve">
+    <value>Option cannot be assigned to the field definition due to the selected type</value>
+  </data>
+  <data name="FieldOptionNotFound" xml:space="preserve">
+    <value>Field option not found</value>
+  </data>
+  <data name="BackToField" xml:space="preserve">
+    <value>Back to field</value>
+  </data>
+  <data name="BackToFields" xml:space="preserve">
+    <value>Back to client fields</value>
+  </data>
+  <data name="ClientFields" xml:space="preserve">
+    <value>Client fields</value>
+  </data>
+  <data name="EditClientField" xml:space="preserve">
+    <value>Edit client field</value>
+  </data>
+  <data name="EditClientFieldOption" xml:space="preserve">
+    <value>Edit field option</value>
+  </data>
+  <data name="NewClientField" xml:space="preserve">
+    <value>New client field</value>
+  </data>
+  <data name="NewClientFieldOption" xml:space="preserve">
+    <value>Add field option</value>
+  </data>
+  <data name="FieldDeletedSuccessfully" xml:space="preserve">
+    <value>Successfully deleted field.</value>
+  </data>
+  <data name="FieldOptionDeletedSuccessfully" xml:space="preserve">
+    <value>Successfully deleted field option.</value>
```

Comment:
suggestion: let's remove the dots at the end of messages (so that it's consistent for all)